deepchem/models/tests/test_generalize.py

```diff
@@ -122,63 +119,65 @@ def model_builder(model_dir):
   assert score > .5
 
 
-#def test_sklearn_classification():
-#  """Test that sklearn models can learn on simple classification datasets."""
-#  np.random.seed(123)
-#  dataset = sklearn.datasets.load_digits(n_class=2)
-#  X, y = dataset.data, dataset.target
-
-#  frac_train = .7
-#  n_samples = len(X)
-#  n_train = int(frac_train*n_samples)
-#  X_train, y_train = X[:n_train], y[:n_train]
-#  X_test, y_test = X[n_train:], y[n_train:]
-#  train_dataset = dc.data.NumpyDataset(X_train, y_train)
-#  test_dataset = dc.data.NumpyDataset(X_test, y_test)
-
-#  classification_metric = dc.metrics.Metric(dc.metrics.roc_auc_score)
-#  sklearn_model = LogisticRegression()
-#  model = dc.models.SklearnModel(sklearn_model)
-
-#  # Fit trained model
-#  model.fit(train_dataset)
-#  model.save()
-
-#  # Eval model on test
-#  scores = model.evaluate(test_dataset, [classification_metric])
-#  assert scores[classification_metric.name] > .5
-
-#def test_sklearn_multitask_classification():
-#  """Test that sklearn models can learn on simple multitask classification."""
-#  np.random.seed(123)
-#  n_tasks = 4
-#  tasks = range(n_tasks)
-#  dataset = sklearn.datasets.load_digits(n_class=2)
-#  X, y = dataset.data, dataset.target
-#  y = np.reshape(y, (len(y), 1))
-#  y = np.hstack([y] * n_tasks)
-#
-#  frac_train = .7
-#  n_samples = len(X)
-#  n_train = int(frac_train*n_samples)
-#  X_train, y_train = X[:n_train], y[:n_train]
-#  X_test, y_test = X[n_train:], y[n_train:]
-#  train_dataset = dc.data.DiskDataset.from_numpy(X_train, y_train)
-#  test_dataset = dc.data.DiskDataset.from_numpy(X_test, y_test)
-
-#  classification_metric = dc.metrics.Metric(dc.metrics.roc_auc_score)
-#  def model_builder(model_dir):
-#    sklearn_model = LogisticRegression()
-#    return dc.models.SklearnModel(sklearn_model, model_dir)
-#  model = dc.models.SingletaskToMultitask(tasks, model_builder)
-
-#  # Fit trained model
-#  model.fit(train_dataset)
-#  model.save()
-#  # Eval model on test
-#  scores = model.evaluate(test_dataset, [classification_metric])
-#  for score in scores[classification_metric.name]:
-#    assert score > .5
+def test_sklearn_classification():
```

Comment:
This modification is not related to this PR. 
However, I think it is better to enable test cases if they pass, and I remove comment out.

CC @rbharath 
Is there any reasons for this comment out?

Reply:
I don't think so! This was cruft left over from previous PRs so good to uncomment